Add DM Home Page (#343)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/whatsnew-10-18-0.md

### Device Management

> #### Fully customizable Home Page

Reviewer: ```suggestion
#### Fully customizable Home page
```

> The Home Page of Device Management is now fully customizable, allowing users to adapt it to their needs and to accommodate any KPI or metric. Being based on the same underlying module as the Cockpit Home Page allows users to experience the same functionality including all available widgets as well as the same look and feel now also in Device Management.

Reviewer: ```suggestion
The Home page of the Device Management application now represents a fully customizable dashboard, which allows users to adapt it to their needs and to accommodate any KPI or metric. Based on the same underlying module as the Cockpit dashboard, users can now experience the same functionality including all available widgets as well as the same look and feel in the Device Management application.
```
